### PR TITLE
Removes plurals library to restore chinese support in lens

### LINKED ISSUE #12: blank page
different computer, all win10  64, same latest chrome ,firefox  version

index page, click search

some  chrome can show,  other  blank page
all firefox blank page
 
zipkin version: docker 2.21.2 2.21.3

```
TypeError: "i is not a function"
    plural core.production.min.js:1
    e core.production.min.js:1
    e core.production.min.js:1
    e core.production.min.js:1
    v core.production.min.js:1
    value core.production.min.js:1
    React 7
    unstable_runWithPriority scheduler.production.min.js:19
    React 4
    Redux 6
    pn traces-action.js:32
react-dom.production.min.js:209:194
    React 5
    unstable_runWithPriority scheduler.production.min.js:19
    React 4
    unstable_runWithPriority scheduler.production.min.js:19
    React 4
    Redux 6
    pn traces-action.js:32
```

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Fixes https://github.com/openzipkin/zipkin-support/issues/12

### CODE REVIEW

File: zipkin-lens/src/translations/zh-cn/messages.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "Address": "",
```
Comment (codefromthecrypt) [inline zipkin-lens/src/translations/zh-cn/messages.json]: @jxc928 @jxc928 @uckyk @hanahmily can some of you help us finish this translation file? It is missing many text

File: zipkin-lens/src/translations/es/messages.json [hunk 2]
```diff
@@ -14,6 +15,7 @@
   "Filter": "Filtro",
   "Limit": "Límite",
   "None": "Ninguno",
+  "One Result": "",
```
Comment (codefromthecrypt) [inline zipkin-lens/src/translations/es/messages.json]: @jeqo @jcchavezs can you help with this and also "{0} Results"?

**Review by anuraaga — APPROVED** [review]

File: zipkin-lens/src/translations/zh-cn/messages.json [hunk 3]
```diff
@@ -14,6 +15,7 @@
   "Filter": "过滤",
   "Limit": "数量",
   "None": "空",
+  "One Result": "",
```
Comment (codefromthecrypt) [inline zipkin-lens/src/translations/zh-cn/messages.json]: @uckyk almost done. can you give for

"One Result"
"{0} Results"

**Comment by uckyk** [comment]
I don't know Whats mean about "one result""{0} results"?
Can you describe it in detail?
Where did this come from? I haven't seen this field

File: zipkin-lens/src/translations/es/messages.json [hunk 4]
```diff
@@ -36,10 +38,10 @@
...
   "Upload JSON": "Subir JSON",
   "View Logs": "Ver Logs",
-  "Archive Trace": "Archivar Traza",
+  "Zipkin": "",
   "Zipkin Home": "Zipkin Inicio",
   "hide annotations": "ocultar anotaciones",
   "show all annotations": "mostrar todas las anotaciones",
   "trace id...": "id de traza...",
-  "{0, plural, one {# Result} other {# Results}}": ""
+  "{0} Results": ""
```
Comment (jeqo) [inline zipkin-lens/src/translations/es/messages.json]:
```suggestion
  "{0} Results": "{0} Resultados"
```

File: zipkin-lens/src/translations/es/messages.json (hunk 2 above)
Comment (jeqo) [reply zipkin-lens/src/translations/es/messages.json]:
```suggestion
  "One Result": "Un Resultado",
```

**Comment by codefromthecrypt** [comment]
@uckyk when you click search on the traces screen

"One result" means one trace returned matching the search parameters
"{0} results" means more than one trace matched the search parameters

the text is new for chinese as never was translated

**Comment by jxc928** [comment]
get it simple
"One result" : "一条"

**Comment by jxc928** [comment]
get it simple

**Comment by jxc928** [comment]
"One result": "一条"
"{0} results": "多条"

**Comment by uckyk** [comment]
got  it . here it is

"One Result" : "1条结果"
"{0} Results"："{0} 条结果"

**Comment by jxc928** [comment]
> got it . here it is
> 
> "One Result" : "1条结果"
> "{0} Results"："{0} 条结果"

seems better.

**Comment by codefromthecrypt** [comment]
Thanks @uckyk @jxc928 @jeqo !


Screenshot 2020-06-04 at 5 14 35 PM
Screenshot 2020-06-04 at 5 15 52 PM
Screenshot 2020-06-04 at 5 16 31 PM
Screenshot 2020-06-04 at 5 17 15 PM

**Pull request merged**